src/analyses/submitContractRequest.ts

```diff
@@ -0,0 +1,28 @@
+import { compileContract } from '../util/compileContract'
+
+
+//TODO: ADD EXTRA PARAM CONTRACTNAME
+export async function submitContractRequest(contractName: string, path: string) {
+
+    const { solName, contractContent, bytecode } = compileContract(path)
+    console.log(solName, 'solName')
+
+    return {
+        "clientToolName": "MythxJS",
```

Comment:
MythxJS should be the default value here (and wherever else clientToolName is set), but it should be configurable; as the library will be used by different tools such as truffle-security, sabre, vscode, etc.